docs/guides/docs/testnet-migration/beta-3-to-beta-4-migration.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Beta 4 Testnet Migration 
+category: Testnet Migration
+parent:
+  label: Guides
+  link: /guides
+---
+
+# Beta 4 Migration
+
+This guide focuses on the transition from the end of tool support for `beta 3` to the start of `beta 4`.
+
+- [Fuelup v0.19.4](https://github.com/FuelLabs/fuelup/releases/tag/v0.19.4)
+- [Sway v0.44.0](https://github.com/FuelLabs/sway/releases/tag/v0.44.0)
+- [TS SDK v0.54.0](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.54.0)
+- [Rust SDK v0.47.0](https://github.com/FuelLabs/fuels-rs/releases/tag/v0.47.0)
+- [Wallet SDK v0.12.2](https://github.com/FuelLabs/fuels-wallet/releases/tag/v0.12.2)
+
+## Fuelup Toolchain
+
+Update to the latest version of `fuelup`:
+
+```sh
+fuelup self update
+```
+
+Install the `beta-4` toolchain:
+
+```sh
+fuelup toolchain install beta-4
+```
+
+Set the `beta-4` toolchain as your default:
+
+```sh
+$ fuelup default beta-4
+default toolchain set to 'beta-4-aarch64-apple-darwin'
+```
+
+## Wallet
+
+Fuel wallet version `0.11.0` is not compatible with beta 4. Please download the non backwards compatible Fuel wallet version `0.12.0` for beta-4 from the Chrome store.
```

Comment:
```suggestion
Fuel wallet version `0.11.0` is not compatible with beta 4. Please download the non backwards compatible Fuel wallet version `0.12.2` for beta-4 from the Chrome store.

```sh
npm install @fuel-wallet@0.12.2
```
```
We should always guide user with commands on how to do the things, it makes it simpler to copy and paste

Reply:
We also need to include a link to install the wallet here, using the zipfile of the latest release.

https://github.com/FuelLabs/fuels-wallet/releases/latest

- Download FuelWallet zip file
- Inside Chrome or Brave;
- Open the extensions page; it can be done by:
- Clicking on settings -> extensions or;
- Accessing brave://extensions/ or chrome://extensions/.
- Enable the "Developer mode" switch on the top right
- Load fuel-wallet.zip by:
- Dragging your downloaded Fuel wallet file and dropping it in the extensions page or;
- Clicking on Load unpacked and selecting the file.
- If all goes right, an onboarding page will instantly open.
The Fuel Wallet extension is now ready to use.